src/main/java/appeng/client/gui/me/common/MEMonitorableScreen.java

```diff
@@ -136,6 +136,38 @@ public MEMonitorableScreen(C container, PlayerInventory playerInventory,
         } else {
             viewCellBg = null;
         }
+
+        if (this.style.isSupportsAutoCrafting()) {
```

Comment:
Just `supportsAutoCrafting` or if the `is` needs to be kept for boolean at least `isSupportingAutoCrafting`?
Looks a bit strange currently.

Reply:
It's just the awful java bean getter naming though :-|